src/routes/project/[projectId]/papers/+page.svelte

```diff
@@ -143,13 +151,37 @@
             {/await}
         </div>
     </div>
-    {#if selectedPaper}
-        <Card.Root class="flex h-full w-[60%] flex-col gap-5 overflow-hidden shadow-lg">
-            <Card.Content>
-                <!-- TODO: replace with paper details card -->
-                <!-- This is done in https://github.com/SE-UUlm/snowballr-frontend/issues/219 -->
-                <pre>{JSON.stringify(selectedPaper.paper!, undefined, 2)}</pre>
-            </Card.Content>
-        </Card.Root>
+    {#if loadingPaper && selectedPaper}
+        <aside
+            class="h-full w-[65%] min-w-75"
+            data-testid="paper-details-card"
+            onClickedOutsideOrEscape={() => {
+                selectedPaper = undefined;
+            }}
+            use:clickOutsideOrEscape
+            transition:fly={{ duration: 100, x: 100, opacity: 0 }}
```

Comment:
```suggestion
            transition:fly={{ duration: 250, x: 150, opacity: 0 }}
```

I tried it with these values. Imo it looked better but felt a bit slow. Maybe something in between?
All the other animation types don't look as good as `fly`.

Reply:
I changed it so a duration of 200ms, what's your opinion on this change?

Reply:
Yes, looks good to me.